channel.py

```diff
@@ -80,6 +80,70 @@ def format_slack_canaryalert(self,canarydrop=None, protocol=settings.PROTOCOL,
         payload['attachments'] = [attachment]
         return payload
 
+    def format_googlechat_canaryalert(self,canarydrop=None, protocol=settings.PROTOCOL,
```

Comment:
nitpick:
is `canarydrop=None` a valid default?

formating: `self,canarydrop=None,` -> `self,<space>canarydrop=None,`